src/js/standard/VolumeSlider/index.js

```diff
@@ -64,7 +64,10 @@ class VolumeSlider extends BaseElement {
         this.on('playerLoadedMetadata', () => this.onVolumeChange());
 
         // Add mouse event listeners
-        this.volumeButton.addEventListener('click', () => this.onClick());
+        if (!window.TouchEvent) {
```

Comment:
How about those laptops that also have touch capabilities?

Reply:
Good point